xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp

```diff
@@ -607,8 +607,31 @@ void CLinuxRendererGLES::RenderUpdateVideo(bool clear, DWORD flags, DWORD alpha)
     CDVDVideoCodecIMXBuffer *buffer = m_buffers[m_iYV12RenderBuffer].IMXBuffer;
     if (buffer != NULL && buffer->IsValid())
     {
-      ManageDisplay();
-      g_IMXContext.SetBlitRects(m_sourceRect, m_destRect);
+      RENDER_STEREO_MODE stereo_mode = g_graphicsContext.GetStereoMode();
+      if (stereo_mode)
+        g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_LEFT);
```

Comment:
Right. The SetStereoView is called after gui rendering here, so it doesn't matter.
So it's just TAB not working, right? What are the symptoms? You didn't include mk01's wrong patch, did you?